<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sevencins</class>
 <widget class="QMainWindow" name="sevencins">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sevencins</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="main">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>741</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QPushButton" name="myShoppingList">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>340</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>My Shopping List</string>
      </property>
     </widget>
     <widget class="QFrame" name="drinkPictureFrame">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>181</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QGraphicsView" name="drinkPicture">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="randomDrinkButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>340</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Shows a random drink. Try something new!</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="text">
       <string>Surprise Me!</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Search for a drink or a specific ingredient.</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showCabinetButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>340</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Shows what is currently in your cabinet.</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="text">
       <string>My Cabinet</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showFavoritesButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>340</y>
        <width>100</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Show your list of favorites.</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="text">
       <string>Favorites</string>
      </property>
     </widget>
     <widget class="QLabel" name="drinkName">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Welcome to 7DC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="drinkInfo">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>361</width>
        <height>211</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="resultsPage">
     <widget class="QPushButton" name="return1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="shoppinglist">
     <widget class="QPushButton" name="return2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="theShoppingList">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>361</width>
        <height>211</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Double click to add to your list:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="ingredientsList">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>191</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="clearShoppingList">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear the list</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.SF NS Text'; font-size:12pt; font-weight:600; color:#fc02ff;&quot;&gt;Your Shopping List&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QToolButton" name="saveIntoTXT">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save to Desktop</string>
      </property>
     </widget>
     <widget class="QLabel" name="liveUpdate">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
